[
  {
    "poster": {
      "poster_name": "Tim Whelan",
      "poster_handle": "timmy"
    },
    "time_stamp": "2021-07-17 06:05:33",
    "edited": {
      "time_stamp": "2021-07-17 06:05:52"
    },
    "text": "Hi all - I am using the `<https://api.hubapi.com/crm/v3/objects/contacts/>` API, passing in a contact Id that I retrieved using the <http://app.HubSpot.com|app.HubSpot.com> portal, but the API returns a 404.  This is happening for approx. half the contacts, the others are returned just fine.\n\nFull Curl example:\n```curl -i -H Content-Type\\:\\ application/json -XGET <https://api.hubapi.com/crm/v3/objects/contacts/787901?archived=false&amp;hapikey=XXXXX>```\nAny ideas what I'm doing wrong?",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2021-07-23 17:13:55"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "Marc Jenkins",
      "poster_handle": "marc.jenkins"
    },
    "time_stamp": "2021-07-17 08:35:31",
    "text": "<@U012W4YEUTX> thank you, it's probably an ADF issue, not HS, but was hoping to find a few other people in my particular boat. ADF wants to pass the entire action in an array for some reason, and HS rejects it of course."
  },
  {
    "poster": {
      "poster_name": "Mohit Prasad",
      "poster_handle": "mohit"
    },
    "time_stamp": "2021-07-17 09:19:16",
    "text": "var request = require(\"request\");\n\nvar options = {\n  method: 'PUT',\n  url: '<https://api.hubapi.com/crm/v3/objects/2-2528946/241337425/associations/2-2518331/242729049/>',\n  qs: {hapikey: 'MY HAPIKEY'},\n  headers: {accept: 'application/json'}\n};\n\nrequest(options, function (error, response, body) {\n  if (error) throw new Error(error);\n\n  console.log(body);\n});"
  },
  {
    "poster": {
      "poster_name": "Mohit Prasad",
      "poster_handle": "mohit"
    },
    "time_stamp": "2021-07-17 09:20:10",
    "text": "<@U012W4YEUTX> This is my request for associating one object with another. But It is giving a response of HTTP 404."
  }
]